src/main/java/pack/controller/IPersonController.java

```diff
@@ -0,0 +1,16 @@
+package pack.controller;
+
+import java.util.List;
+
+import pack.model.Person;
+
+
+public interface IPersonController {
+	public List<Person> getPersons();
```

Comment:
Usually Controllers don't need an interface, as they are not used in other components/